templates/repo/diff/compare.tmpl

```diff
@@ -60,7 +64,15 @@
         	<div class="ui info message show-form-container">
         		<button class="ui button green show-form">{{.i18n.Tr "repo.pulls.new"}}</button>
         	</div>
-        	{{end}}
+        	{{ else }}
+        		<div class="ui warning message">
+        			{{if .Issue.IsPull}}
+        				{{.i18n.Tr "repo.archive.title"}}
+        			{{else}}
+        				{{.i18n.Tr "repo.archive.title"}}
```

Comment:
It seems the two titles are the same?

Reply:
you right, this does not belong there, missed that, sorry

Reply:
should be fixed